examples/js/effects/VREffect.js

```diff
@@ -399,12 +399,12 @@ THREE.VREffect = function ( renderer, onError ) {
 
 			return;
```

Comment:
Ah, I think you're right. The improvements I saw must have been some other fixes I did (remove unnecessary clears). I'll take another look and make a new PR.